client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/item-row.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import TextField from 'woocommerce/woocommerce-services/components/text-field';
+import {
+	setCustomsItemDescription,
+	setCustomsItemTariffNumber,
+	setCustomsItemOriginCountry,
+} from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+import CountryDropdown from 'woocommerce/woocommerce-services/components/country-dropdown';
+
+const TariffCodeTitle = localize( ( { translate } ) =>
+	<span>{ translate( 'Tariff Code' ) } (<a href="https://hts.usitc.gov/"
```

Comment:
I think this should be "Tariff code" for consistency with other labels. Also, this link and the ITN more info link should be `ExternalLink`s so that people are prepared to be taken out of the app.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/24974/commits/edfa0f3f5c5c827a09fc7d74387af926badb0362